### PR TITLE
initial pass on tests for collocated journals

### PR DESCRIPTION
**Pull request by alfredodeza**

### CODE REVIEW

**Comment by alfredodeza** [comment]
Let's not merge this until we have the other scenarios enabled that actually test this PR: https://github.com/ceph/ceph-build/pull/570

File: tests/functional/centos/7/journal-collocation/scenario.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
     },
     'osd0': {
         'username': 'vagrant',
-        'components': [],
+        'components': ['collocated_journals'],
+        'devices': ['/dev/sda', '/dev/sdb'],
```
Comment (andrewschoen) [inline tests/functional/centos/7/journal-collocation/scenario.py]: It's a bummer that we have to list devices here and in the group_vars, but I'm not sure how to get around that.

**Comment by alfredodeza** [comment]
jenkins test ansible2.2-journal-collocation

**Comment by alfredodeza** [comment]
jenkins test ansible2.2-journal-collocation

File: tox.ini [hunk 2]
```diff
@@ -17,19 +17,19 @@ deps=
   -r{toxinidir}/tests/requirements.txt
 changedir=
   # test a 3 mon cluster and ensures ceph.conf is rendered correctly
-  xenial-conf-tests: {toxinidir}/tests/functional/ubuntu/16.04/mon
+  xenial_conf_tests: {toxinidir}/tests/functional/ubuntu/16.04/mon
   # tests a 1 mon 1 osd xenial cluster using raw_multi_journal OSD scenario
-  xenial-mon-osd: {toxinidir}/tests/functional/ubuntu/16.04/mon-osd
+  xenial_mon_osd: {toxinidir}/tests/functional/ubuntu/16.04/mon_osd
```
Comment (andrewschoen) [inline tox.ini]: The directories should not have changed here, only the environment name

**Comment by andrewschoen** [comment]
jenkins test this please

**Pull request merged**